typings/index.d.ts

```diff
@@ -299,7 +299,7 @@ export interface RenderOptions {
 }
 
 type Debug = {
-  (message?: string);
+  (message?: string): any;
```

Comment:
`void` would be a proper return type here, what do you think?
```suggestion
  (message?: string): void;
```

Reply:
Agreed! Will commit the suggestion, thank you 👍🏻